### PR TITLE
Issue-6203 Add `go.get("#sprite", "animation")`

### LINKED ISSUE #6203: Add go.get("#sprite", "current_animation") getter
**Is your feature request related to a problem? Please describe (REQUIRED):**
There is no any way to get current animation from GO sprite label. If I make layout in go collection files and I need for some reason get image from sprite (to clone in with particles for example), I can't make it.

**Describe the solution you'd like (REQUIRED):**
Add getter for current animation

**Describe alternatives you've considered (REQUIRED):**
For now current solution is to make the data with duplicated name of sprites of every GO in collection

### PR DESCRIPTION
**Pull request by AGulev**
fix https://github.com/defold/defold/issues/6203

### CODE REVIEW

File: engine/gamesys/src/gamesys/components/comp_sprite.cpp [hunk 1]
```diff
@@ -111,6 +111,7 @@ namespace dmGameSystem
 
     static const dmhash_t SPRITE_PROP_CURSOR = dmHashString64("cursor");
     static const dmhash_t SPRITE_PROP_PLAYBACK_RATE = dmHashString64("playback_rate");
+    static const dmhash_t SPRITE_PROP_ANIM_ID = dmHashString64("animation_id");
```
Comment (JCash) [inline engine/gamesys/src/gamesys/components/comp_sprite.cpp:114]: For consistency with gui, I prefer "animation"
https://defold.com/ref/stable/gui/#gui.get_flipbook:node

Comment (AGulev) [reply engine/gamesys/src/gamesys/components/comp_sprite.cpp:114]: Do you mean `animation` instead of `animation_id` ?

Comment (AGulev) [reply engine/gamesys/src/gamesys/components/comp_sprite.cpp]: fixed

File: engine/gamesys/src/gamesys/components/comp_sprite.cpp [hunk 2]
```diff
@@ -1096,6 +1097,11 @@ namespace dmGameSystem
             out_value.m_Variant = dmGameObject::PropertyVar(GetPlaybackRate(component));
             return dmGameObject::PROPERTY_RESULT_OK;
         }
+        else if (get_property == SPRITE_PROP_ANIM_ID)
```
[the remark this answers is not in the document]
Comment (AGulev) [reply engine/gamesys/src/gamesys/components/comp_sprite.cpp]: makes sense, fixed

**Review by JCash — APPROVED** [review]
lgtm

**Pull request merged**